types/browser-sync/index.d.ts

```diff
@@ -1,10 +1,11 @@
-// Type definitions for browser-sync 2.26
+// Type definitions for browser-sync 2.27
 // Project: http://www.browsersync.io/
 // Definitions by: Asana <https://asana.com>,
 //                 Joe Skeen <https://github.com/joeskeen>
 //                 Thomas "Thasmo" Deinhamer <https://thasmo.com/>
 //                 Kiyotoshi Ichikawa <https://github.com/aznnomness>
 //                 Yuma Hashimoto <https://github.com/yuma84>
+//                 Andreas Bomholtz <https://github.com/seluxit>
```

Comment:
Note that this adds you as an owner of the package, will ping you for reviews, etc. If you're just looking for contribution credit, we use the git history to manage that, so you don't need to provide this if you don't want to become an owner.